Clarify the use of the Lookup API (#12088)
apache/druid: docs/querying/lookups.md

### Update lookup

> A `POST` to a particular lookup extractor factory via `/druid/coordinator/v1/lookups/config/{tier}/{id}` will create or update that specific extractor factory.

Reviewer: ```suggestion
A `POST` to a particular lookup extractor factory via `/druid/coordinator/v1/lookups/config/{tier}/{id}` creates or updates that specific extractor factory.
```

Reviewer: Really minor nit suggestions that replace future tense with present.

This will replace the `site_id_customer1` lookup in the `realtime_customer1` with the definition above.

> Make sure to assign a unique version identifier each time you update a lookup extractor factory. If the version already exists, the call will fail.

Reviewer: ```suggestion
Make sure to assign a unique version identifier each time you update a lookup extractor factory. If the version already exists, the call falls.
```

> Make sure to assign a unique version identifier each time you update a lookup extractor factory. If the version already exists, the call will fail.

Reviewer 2: ```suggestion
Assign a unique version identifier each time you update a lookup extractor factory. Otherwise the call will fail.
```
